go.sum

```diff
@@ -46,6 +46,8 @@ github.com/docker/go-units v0.3.3 h1:Xk8S3Xj5sLGlG5g67hJmYMmUgXv5N4PhkjJHHqrwnTk
 github.com/docker/go-units v0.3.3/go.mod h1:fgPhTUdO+D/Jk86RDLlptpiXQzgHJF7gydDDbaIK4Dk=
 github.com/firecracker-microvm/firecracker-go-sdk v0.15.2-0.20190410183546-4f11ac73a965 h1:QKioOzqmg7DY/YDhGLYZsP8bq2T3BahVHdyPsf0uq/E=
 github.com/firecracker-microvm/firecracker-go-sdk v0.15.2-0.20190410183546-4f11ac73a965/go.mod h1:QcNsz2gYkcTI/zAQl3dYeLwf7KxtjKnt7aGklhn9yYk=
+github.com/firecracker-microvm/firecracker-go-sdk v0.15.2-0.20190627223500-b2e8284e890c h1:jAAxKV5SD/y2nNKNLHgAUmfgbrvfN4xBiKR+b2qK2pM=
```

Comment:
nit: We should probably run `go mod tidy` as it looks like there's still checksums in here for the older version of the firecracker-go-sdk.

Reply:
Fixed!